interfaces/cython/cantera/examples/onedim/UnscaleVariances.py

```diff
@@ -0,0 +1,655 @@
+## OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+## LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+## NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+## EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.? THE UNIVERSITY OF DAYTON
+## HAS NO OBLIGATION TO SUPPORT THE SOFTWARE.
+#################################################################################################
+
+"""
+Please cite this work as follows:
+Briones, A.M., Olding, R., Sykes, J.P., Rankin, B.A., McDevitt, K., Heyne, J.S., 
+"Combustion Modeling Software Development, Verification and Validation," Proceedings of the 2018 ASME Power & Energy Conference,
+PowerEnergy2018-7433, Lake Buena Vista, FL. 
+"""
+
+import sys
+import getopt
+import numpy as np
+from scipy.interpolate import interp1d
+from matplotlib import cm
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import axes3d,Axes3D
+from multiprocessing import Process, Pipe
+from collections import OrderedDict
+
+# UDRI imports
+from table_def import TableDef
+from Utils import PrintFlush
+from Graphics import PlotDlg
+
+# Input FileHandler class - responsible for reading input file 
+class InputFileHandler():
+
+    def __init__(self, fileName):
+        self._fileName = fileName
+        self._numGridpoints = 0     # num fmean
+        self._numFlamelets = 0      # num cmean
+        self._numFvars = 0          # num fvar 
+        self._numVars = 0           # num variables 
+        self._numCvars = 1          # num of cvar
+        self._numSpecies = 0        # num of species
+        self._numQoi = 0            # num QOI
+        self._columnHeader = "none" # column headings
+        self._data = []             # table data
+        self._input = open(fileName,'r')
+        self._variableDict = OrderedDict()
+
+    @property
+    def numGridpoints(self):
+        '''number of fmean values'''
+        return self._numGridpoints
+
+    @property
+    def numFlamelets(self):
+        '''number of cmean values'''
+        return self._numFlamelets
+
+    @property
+    def numFvars(self):
+        '''number of fvar values'''
+        return self._numFvars
+
+    @property
+    def numVars(self):
+        '''number of variables'''
+        return self._numVars
+
+    @property
+    def numCvars(self):
+        '''number of Cvar values'''
+        return self._numCvars
+
+    @property
+    def numSpecies(self):
+        '''number of species'''
+        return self._numSpecies
+
+    @property
+    def numQoi(self):
+        '''number of Qoi values'''
+        return self._numQoi
+
+    @property
+    def columnHeader(self):
+        '''column headings'''
+        return self._columnHeader
+
+    @property
+    def data(self):
+        '''table data'''
+        return self._data
+
+
+    # reads in a data set header and keeps track of number of lines read

```

Comment:
Python functions should be documented with docstrings, not comments. Also, there are too many blank lines here (one is sufficient) - please avoid adding extraneous whitespace